reconfiguration/cmf/concord.cmf

```diff
@@ -117,6 +117,7 @@ Msg AddRemoveWithWedgeStatusResponse 31 {
 
 Msg UnwedgeStatusRequest 32{
     uint64 sender
+    bool bft
```

Comment:
not too self explanatory flag

Reply:
This will break APIs for the product, I would prefer to change it in a future PR

Reply:
what does it mean ?

Reply:
fixed